kickstart/PXELinux_atomic.erb

```diff
@@ -0,0 +1,18 @@
+<%#
+kind: PXELinux
+name: Atomic PXE
+description: Start Atomic from the bare metal (installer) image.
+oses:
+- CentOS-Atomic 7
+- Fedora-Atomic 22
+- RedHat-Atomic 7
+%>
+
+DEFAULT pxeboot
```

Comment:
repo= seems like a good thing to put behind a conditional based on the name, yep.  I don't understand why the vmlinuz/initrd would be different though, why's that?  Shouldn't Foreman manage those from the installation medium?